src/cli/commands/mod.rs

```diff
@@ -55,10 +58,13 @@ enum Commands {
 )]
 #[command(propagate_version = true)]
 struct IPCAgentCliCommands {
+    // If provided, outputs the completion file for given shell
+    #[arg(long = "generate", value_enum)]
```

Comment:
Should we call it `install_autocompletion` or `autocompletion`? Generate seems like too general

Reply:
I guess this is the convention from clap?